doc/time/in.rdoc

```diff
@@ -1,7 +1,3 @@
-- <tt>in: zone</tt>: a timezone _zone_, which may be:
-  - A string offset from UTC.
-  - A single letter offset from UTC, in the range <tt>'A'..'Z'</tt>,
-    <tt>'J'</tt> (the so-called military timezone) excluded.
-  - An integer number of seconds.
-  - A timezone object;
-    see {Timezone Argument}[#class-Time-label-Timezone+Argument] for details.
+- <tt>in: zone</tt>: a timezone +zone+.
```

Comment:
It seems like there's only one place (in `Time.at` in `timev.rb`) that uses this file. I think we can just remove this file and directly embed the contents.

Reply:
> It seems like there's only one place (in `Time.at` in `timev.rb`) that uses this file. I think we can just remove this file and directly embed the contents.

We can, but I kinda like the parallel treatment.